src/openvic-simulation/politics/Rule.cpp

```diff
@@ -183,16 +191,6 @@ bool RuleManager::setup_rules(BuildingTypeManager const& building_type_manager)
 		}
 	}
 
-	for (std::string const& type : building_type_manager.get_building_type_types()) {
```

Comment:
Why remove this?
Now we lack naval_base & fort rules.

Reply:
Vic2 doesn't support naval_base and fort building rules and has no localisation for them.
Although the fort and naval base button tooltips say "Economic policy allows building" similar to the railroad button, adding `build_fort = no` or `build_naval_base = no` to an economic policy has no effect - they don't show up in the tooltip at all, not even as unlocalised rules.

Reply:
what about the extra buildings that mods add?